### ISSUE TITLE
Recovering of account was weird

### ISSUE THREAD

**Issue by AlexanderTserkovniy** [MetaMask/metamask-extension#1743]
Hi guys,

I thought that I completely lost an access to the account, but it managed somehow to restore and it seems like it is not expected behaviour.

So I started from I forgot password link and entered my 12 words, but got "seed phrase is invalid" so I was surprised and a bit shocked, how is that possible. So I did it 10-20 times and it did not work. Okay, I just tried next thing. I replaced first and the last letter with the same letter and BOM, it worked! Account just unblocked.

But this is not it, I noticed that among 3 of my accounts I see only 1, which is 0. So I was frustrated, maybe I recovered everything incorrectly, but when I pressed plus sign, my second account appeared in state as it was, which is WOW, I have not expected that.

Please take a closer look and fix some stuff.

Thanks.

**Comment by tmashuang** [comment]
The add accounts for existing accounts is a related issue, #654. I am curious to what you mean by `I replaced first and the last letter with the same letter`?

**Comment by AlexanderTserkovniy** [comment]
I have 12 words, let's say `bla bla2 bla.... bla12bla`, so I replaced first `b` with the same `b` and last `a` with the same `a` and it worked.

**Comment by tmashuang** [comment]
Interesting, this is a rare occurrence. I can't seem to reproduce this. Did the first initial input a copy and paste of the seed phrase? I know the seed phrase has to be pretty precise, only allowing for one space in between the words and all lowercase.

**Comment by AlexanderTserkovniy** [comment]
Was a copy right, maybe source issue, but still an issue.

**Comment by kumavis** [comment]
what im reading from this issue:
- may need to sanitizing seed input
- seed phrase accounts is confusing, and maybe the non-empty account search is not working

**Comment by Papa-Ron** [comment]
Hello, I'm hoping it's not too late to get in on this conversation.
I also cannot get my seed recovery to work on a new computer, and am hoping to find out what to do next,  it makes it hard when one can't find a support email to ask these things,

Thanks, Ron

**Comment by flyswatter** [comment]
Sorry, apparently we hadn't added our new support page to our main site yet. (Adding it now](https://github.com/MetaMask/landing/pull/54).

Here is the link, by the way:
http://metamask.consensyssupport.happyfox.com/kb/

Can you describe the issue as you're experiencing it, @Papa-Ron? In this original note, it was first retyped wrong (probably) a few times, and also there was confusion about how to re-generate the previous accounts. Is this what happened to you? Is your seed phrase not being recognized?

Maybe try out this tool I made, which is a little more liberal with how it tries to recognize a seed word, and only requires the first four letters of each word.
https://flyswatter.github.io/js-recover-bip39/
Since you shouldn't type your seed phrase into anything you can't read the source code of, you can also read its source code here:
https://github.com/flyswatter/js-recover-bip39/blob/master/index.js

**Comment by Papa-Ron** [comment]
Wow,

I'm very impressed that you Dan Finlay, would take the time to answer my message, and on a Saturday no less, so let me start by saying thank you, I realize you are probably busy all the time so I hope I'm not adding to that pressure.


I've really got a couple issues that I can't resolve and I'm sure my age and lack of computer skills are part of it, so here's the story, I'll be quick.


In the middle of the Veritaseum craze I purchased some tokens, opened a MetaMask account and a EtherDelta account to get them.

I then got a new computer, I exported my EtherDelta account okay, only to find all of my small balance of Ethereum gone and half of my Veri, maybe they needed more ether gas so they couldn't take it all, now I'm getting off track, sorry


But putting in my seeds to export my MetaMask account didn't work, but I will try to follow your email that you sent to see if it woroks


But what was weird is I could never send my Veri over to Meta, so I just kept it in EtherDelta, my mistake, I'm not sure why I couldn't get it over there, I'll try again, hoping to get the rest of whats lefty transferred, I'm guessing that there is now way to track down the Thief??


Thanks for listening, Ron




________________________________

**Comment by flyswatter** [comment]
Hi Ron,

Thanks for valuing my time, I wish I had more of it to give.

First of all, you're not the first user to be confused about whether their EtherDelta account was backed up, or where their funds went. This is because EtherDelta allows you to create local in-site accounts *in addition to* MetaMask accounts, and so I've seen many users think their accounts were backed up with their MetaMask seed phrases, just to find out they had used an EtherDelta account locally instead.

That said, you say you exported your EtherDelta account okay. I assume you mean the "private key". If you did that, and you have your metamask seed phrase, I don't see why you shouldn't be able to recover your VERI tokens.

The solution is a little weird, since I'm not 100% sure whether you bought your VERI with your EtherDelta account or with your MetaMask account. It might make sense that you bought some with each, and that could explain why you don't see your full balance?

A few things you can do now:
1. Keep trying to import your MetaMask seed phrase, maybe you've misspelled a word somewhere, in case you bought the tokens with VERI.
2. In EtherDelta, you should be able to import the private key that you backed up from EtherDelta before. Once you do that, do you see your VERI tokens in EtherDelta? Keep in mind that there are two ways you can have a token: Either in your account, or deposited in EtherDelta itself. It's possible you just still have some tokens on the exchange. I'd be surprised if any thief were involved.

Feel free to open a ticket on our support system, it's a more private place to discuss these things, like if you wanted to tell us the account number so we could look up the balance, you might rather do that in private, so strangers can't see.

Good luck!

**Comment by flyswatter** [comment]
Closing for now as a duplicate of #654.
Additionally, we should continuously try to make seed word recovery more flexible.

**Comment by Papa-Ron** [comment]
Dan,

Received 2 emails from you folks, both saying that the support ticket is closed, something about it being weird, sorry to hear that


But good news, you we're right I did have a word spelled wrong, it was fall, I kept seeing fail, no worries, I'm back in, and everything is there, thank, you, Ron


________________________________

**Comment by flyswatter** [comment]
Great to hear Ron,

Sorry the closed issue messages were confusing. It wasn't your ticket that we closed, it was this specific issue, the difference is just specific to our internal process, a ticket is a person with a problem, an issue is a change we're going to make to our software.

Anyways, glad you got that sorted out!

**Comment by Papa-Ron** [comment]
Yeah, I'm thrilled to death and all my coins are there


Hey we've got a youtube channel, with about 6000 subs, all on Crypto, we ought to do a Skype interview with you going over MetaMask, and maybe a few pointers, it could be valuable to all the new folks, please consider it, thanks Ron


Our Channel; https://www.youtube.com/channel/UCVvDOPn2zeBBd_07Y_-Hxtg

[https://yt3.ggpht.com/-5FGWHapDUVU/AAAAAAAAAAI/AAAAAAAAAAA/Fn7mXYj12UE/s100-c-k-no-mo-rj-c0xffffff/photo.jpg]<https://www.youtube.com/channel/UCVvDOPn2zeBBd_07Y_-Hxtg>

Crypto-Bit Brothers - YouTube<https://www.youtube.com/channel/UCVvDOPn2zeBBd_07Y_-Hxtg>
www.youtube.com
Once exposed to the world of Crypto, we realized how outdated and corrupt the banking system is, and decided to jump on this fast moving adventure to secure ...




________________________________

**Comment by donkthought** [comment]
Hey gentlemen, 
 I've been having an issue moving my account over to a different computer and Metamask is not accepting my seed phrase. It's registering as invalid. Might you be able to help?

**Comment by johnjames333** [comment]
@donkthought 
@AlexanderTserkovniy  
we have developed our own hardware for cracking Metamask wallet password with superfast speed
and if you still has problem I can help you run it. you can find my contact details on my profile
or you can send me message in telegram: @johnjames333

**Comment by richardsonn09** [comment]
@AlexanderTserkovniy 
@donkthought 
If you couldn't restore your lost fund, I can recover your wallet if you want need help send me email : anywalletrecovery@gmail.com
